marklogic-data-hub/src/test/resources/pii-test/hub-internal-config/databases/trace-database.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Since @akshaysonvane is currently working on merging jobs and tracing, it will be interesting to see whether this test fares ok.  We should retain backward compatibility with his change.